src/wazuh_db/wdb_agent.c

```diff
@@ -138,19 +155,19 @@ int wdb_update_agent_version(int id, const char *os_name, const char *os_version
             result = 1;
             break;
         case OS_INVALID:
-            mdebug1("GLobal DB Error in the response from socket");
+            mdebug1("Global DB Error in the response from socket");
             mdebug2("Global DB SQL query: %s", wdbquery);
             break;
         default:
-            mdebug1("GLobal DB Cannot execute SQL query; err database %s/%s.db", WDB2_DIR, WDB2_GLOB_NAME);
+            mdebug1("Global DB Cannot execute SQL query; err database %s/%s.db", WDB2_DIR, WDB2_GLOB_NAME);
             mdebug2("Global DB SQL query: %s", wdbquery);
             result = OS_INVALID;
     }
 
     return result;
 }
 
-/* Update agent's last keepalive time. Returns OS_SUCCESS or -1 on error. */
+/* Update agent's last keepalive time. Sends a request to Wazuh-DB. Returns OS_SUCCESS or -1 on error. */
 int wdb_update_agent_keepalive(int id) {
```

Comment:
Shouldn't this method also set the agent `sync_status` to `WDB_UNSYNCED`?

Reply:
This is a good question

Reply:
It's true. This method must change sync_status.